From 9e8592849675baa11a30774550a24b34d5611b1b Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 14 Mar 2019 12:41:01 +0000
Subject: [PATCH 02/13] drop ARMv7 special handling for console port

It's unnecessary and breaks grpahical output. Qemu generates a DT with all
the appropriate bits in it to setup the console.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 oz/Guest.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/oz/Guest.py b/oz/Guest.py
index 034749d..e346904 100644
--- a/oz/Guest.py
+++ b/oz/Guest.py
@@ -481,8 +481,6 @@ class Guest(object):
         if initrd:
             oz.ozutil.lxml_subelement(osNode, "initrd", initrd)
         if cmdline:
-            if self.tdl.arch == "armv7l":
-                cmdline += " console=ttyAMA0"
             oz.ozutil.lxml_subelement(osNode, "cmdline", cmdline)
         if self.tdl.arch == "aarch64":
             loader, nvram = oz.ozutil.find_uefi_firmware(self.tdl.arch)
-- 
2.20.1

